### PR TITLE
chore(*): Roadmap Update Milestone 16

### PR DESCRIPTION
**Pull request by radomirchev**
Closes #  

### Additional information (check all that apply):
 - [ ] Bug fix
 - [ ] New functionality
 - [x] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -2,6 +2,29 @@
...
+
+1. Accordion component
+2. Scheduler component
+3. Grid Cell Merging
+
+# Previous Milestone
+
 ## Milestone 15 (Due by February, 2021)
 
 1. Filters should support filtering by formatted value [#8009](https://github.com/IgniteUI/igniteui-angular/issues/8009)
```
Comment (kdinev) [inline ROADMAP.md:30]: Mark the **DONE** items from Milestone 15.

**Review by kdinev — CHANGES_REQUESTED** [review]

File: ROADMAP.md (hunk 1 above)
Comment (radomirchev) [reply ROADMAP.md:30]: Updated

Comment (kdinev) [inline ROADMAP.md:30]: Filtering by formatted value and the percent and currency type columns are still not marked as done in milestone 15.

**Review by kdinev — CHANGES_REQUESTED** [review]

File: ROADMAP.md (hunk 1 above)
Comment (radomirchev) [reply ROADMAP.md]: When you stage but don't push...

**Review by kdinev — APPROVED** [review]

**Pull request merged**